src/db/nodeDb.js

```diff
@@ -122,7 +134,7 @@ module.exports = {
       error_messages: readResult.cachedValue.errorMessages
     };
 
-    await nodeDb('states').insert(entry).onConflict(['contract_tx_id', 'sort_key']).ignore();
+    await nodeDb('states').insert(entry).onConflict(['contract_tx_id']).merge();
```

Comment:
will the trigger be also..triggered on merge?

Reply:
yes